avro-serializer/src/main/java/io/confluent/kafka/serializers/subject/SubjectNameStrategy.java

```diff
@@ -19,10 +19,14 @@
 import io.confluent.common.Configurable;
 
 /**
+ * {@Deprecated} as of 4.1.3 use
+ * {@link io.confluent.kafka.serializers.subject.v2.SubjectNameStrategy}.
```

Comment:
The `@deprecated` JavaDoc annotation should go last in the JavaDoc and should be of the form:

```
 * @deprecated use {@link io.confluent.kafka.serializers.subject.v2.SubjectNameStrategy} instead.
```